spdlog/src/logger.rs

```diff
@@ -115,6 +115,34 @@ pub struct Logger {
     periodic_flusher: Mutex<Option<(Duration, PeriodicWorker)>>,
 }
 
+impl Debug for Logger {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("Logger")
+            .field("name", &self.name)
+            .field("level_filter", &self.level_filter())
+            .field(
+                "sinks",
+                &self
+                    .sinks
+                    .iter()
+                    .map(|sink| sink.level_filter())
+                    .collect::<Vec<_>>(),
+            )
```

Comment:
```suggestion
            )
            .field("flush_level_filter", &self.flush_level_filter())
            .field(
                "error_handler",
                if self.error_handler.read().is_some() {
                    &"Some"
                } else {
                    &"None"
                },
```

Also outputs for `flush_level_filter` and `error_handler` fields.